open-metadata-implementation/adapters/open-connectors/repository-services-connectors/audit-log-connectors/audit-log-log4j-connector/pom.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!-- SPDX-License-Identifier: Apache-2.0 -->
+<!-- Copyright Contributors to the ODPi Egeria project. -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <parent>
+        <artifactId>audit-log-connectors</artifactId>
+        <groupId>org.odpi.egeria</groupId>
+        <version>1.3-SNAPSHOT</version>
+    </parent>
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <name>Audit Log Log4J Connector</name>
+    <description>
+        Provides a log4j audit log destination for the Open Metadata Repository Services (OMRS).
```

Comment:
technically isn't this using slf4j not log4J ? (though slf can then be configured to use log4J)

Reply:
followup in #2322